Week1/js-exercises/Exercise-6.js

```diff
@@ -0,0 +1,13 @@
+//  ===EXERCISE 6===
+
+'use strict';
+
+console.log('EXERCISE 6 - Log an array of animals');
+
+let myArray = [];
+console.log('[]');
+console.log(myArray);
+let animals = ['lion', 'rabbit', 'gazelle'];
+console.log(animals);
+animals.push('Piglet');
```

Comment:
small nit: `Piglet` starts with an upper case letter where the other animals start with lower case.